tests/core/gadget-update-pc/task.yaml

```diff
@@ -141,10 +142,9 @@ execute: |
 
     if [[ "$SPREAD_REBOOT" == 0 ]]; then
 
-        p=$(new_snap_declaration "$BLOB_DIR" pc_x2.snap --snap-decl-json decl-headers.json)
```

Comment:
just to be clear, we don't need this call because there is no change to the snap-declaration assertion for this new snap revision so that's why this is okay to leave out - and indeed we only have one single decl-headers.json too so the new assertion couldn't meaningfully change from the previous revision of the snap-declaration assertion anyways

Reply:
Yes, there's no changes to snap declaratios, so all we need are just new snap revision assertions.